emcc.py

```diff
@@ -1731,6 +1731,9 @@ def setup_pthreads(target):
 
   default_setting('DEFAULT_PTHREAD_STACK_SIZE', settings.STACK_SIZE)
 
+  if settings.EMBIND:
+    settings.EXPORTED_FUNCTIONS.append('__embind_initialize_bindings')
```

Comment:
Isn't it nicer to leave the keepalive in the source, rather than add lines to `emcc.py`? Leaving the annotation in the various source files spreads it out rather than centralize all the libraries' quirks in `emcc.py`, I think.

Reply:
Hmm.. perhaps yes, but this function only needs to be exported in the pthread case.. so it was being exported unnecessarily in the non-threaded case.

Perhaps I can make that work by putting `ifdef` in the source code instead.

Reply:
I see. Yeah, ifdef in the source sounds better if it works.

Reply:
The dependency here is from `worker.js -> __embind_initialize_bindings`  but sadly we don't have way to declare worker.js dependencies except here in `emcc.py`.

I looked into making a separate multi-threaded version libembind would be needed to make that happen.. but that seemed more complex that just add the dependency here in emcc.py.

I improved it by gouping adding this to the existing `worker_imports` list to its very explict.